app/javascript/packages/components/icon.tsx

```diff
@@ -0,0 +1,296 @@
+  | 'expand_more'
+  | 'facebook'
+  | 'fast_forward'
+  | 'fast_rewind'
+  | 'favorite'
+  | 'favorite_border'
+  | 'file_download'
+  | 'file_present'
+  | 'file_upload'
+  | 'filter_alt'
+  | 'filter_list'
+  | 'fingerprint'
+  | 'first_page'
+  | 'flag'
+  | 'flickr'
+  | 'flight'
+  | 'flooding'
+  | 'folder'
+  | 'folder_open'
+  | 'format_quote'
+  | 'format_size'
+  | 'forum'
+  | 'github'
+  | 'grid_view'
+  | 'group_add'
+  | 'groups'
+  | 'hearing'
+  | 'help'
+  | 'help_outline'
+  | 'highlight_off'
+  | 'history'
+  | 'home'
+  | 'hospital'
+  | 'hotel'
+  | 'hourglass_empty'
+  | 'hurricane'
+  | 'identification'
+  | 'image'
+  | 'info'
+  | 'info_outline'
+  | 'insights'
+  | 'instagram'
+  | 'keyboard'
+  | 'label'
+  | 'language'
+  | 'last_page'
+  | 'launch'
+  | 'lightbulb'
+  | 'lightbulb_outline'
+  | 'link'
+  | 'link_off'
+  | 'list'
+  | 'local_cafe'
+  | 'local_fire_department'
+  | 'local_gas_station'
+  | 'local_grocery_store'
+  | 'local_hospital'
+  | 'local_laundry_service'
+  | 'local_library'
+  | 'local_offer'
+  | 'local_parking'
+  | 'local_pharmacy'
+  | 'local_police'
+  | 'local_taxi'
+  | 'location_city'
+  | 'location_on'
+  | 'lock'
+  | 'lock_open'
+  | 'lock_outline'
+  | 'login'
+  | 'logout'
+  | 'loop'
+  | 'mail'
+  | 'mail_outline'
+  | 'map'
+  | 'masks'
+  | 'medical_services'
+  | 'menu'
+  | 'military_tech'
+  | 'more_horiz'
+  | 'more_vert'
+  | 'my_location'
+  | 'navigate_before'
+  | 'navigate_far_before'
+  | 'navigate_far_next'
+  | 'navigate_next'
+  | 'near_me'
+  | 'notifications'
+  | 'notifications_active'
+  | 'notifications_none'
+  | 'notifications_off'
+  | 'park'
+  | 'people'
+  | 'person'
+  | 'pets'
+  | 'phone'
+  | 'photo_camera'
+  | 'print'
+  | 'priority_high'
+  | 'public'
+  | 'push_pin'
+  | 'radio_button_unchecked'
+  | 'rain'
+  | 'reduce_capacity'
+  | 'remove'
+  | 'report'
+  | 'restaurant'
+  | 'rss_feed'
+  | 'safety_divider'
+  | 'sanitizer'
+  | 'save_alt'
+  | 'schedule'
+  | 'school'
+  | 'science'
+  | 'search'
+  | 'security'
+  | 'send'
+  | 'sentiment_dissatisfied'
+  | 'sentiment_neutral'
+  | 'sentiment_satisfied'
+  | 'sentiment_satisfied_alt'
+  | 'sentiment_very_dissatisfied'
+  | 'settings'
+  | 'severe_weather'
+  | 'share'
+  | 'shield'
+  | 'shopping_basket'
+  | 'snow'
+  | 'soap'
+  | 'social_distance'
+  | 'sort_arrow'
+  | 'spellcheck'
+  | 'star'
+  | 'star_half'
+  | 'star_outline'
+  | 'store'
+  | 'support'
+  | 'support_agent'
+  | 'text_fields'
+  | 'thumb_down_alt'
+  | 'thumb_up_alt'
+  | 'timer'
+  | 'toggle_off'
+  | 'toggle_on'
+  | 'topic'
+  | 'tornado'
+  | 'translate'
+  | 'trending_down'
+  | 'trending_up'
+  | 'twitter'
+  | 'undo'
+  | 'unfold_less'
+  | 'unfold_more'
+  | 'update'
+  | 'upload_file'
+  | 'verified'
+  | 'verified_user'
+  | 'visibility'
+  | 'visibility_off'
+  | 'volume_off'
+  | 'warning'
+  | 'wash'
+  | 'wifi'
+  | 'work'
+  | 'youtube'
+  | 'zoom_in'
+  | 'zoom_out'
+  | 'zoom_out_map';
+
```

Comment:
Might be nice to have this list live in another file, just for the sake of readability.

Reply:
Hm, that's a good thought. I guess the question is... where to put it?

One consideration is that I'm hoping to remove the "Custom Icon" implementation mixed in here (`createIconComponent`), since the one icon being used is available through this new icon set (`photo_camera`), which will hopefully make the file a bit smaller overall. 

I think I'll plan to merge this as-is, but follow-up with that work, and if we decide there's a better place for the icon list type, I can plan to include it there.

Reply:
Sounds good!